### PR TITLE
Add a simple command to uplc to dump the cost model params

### PR DESCRIPTION
**Pull request by michaelpj**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: plutus-core/executables/uplc/Main.hs [hunk 1]
```diff
@@ -212,6 +219,11 @@ runConvert (ConvertOptions inp ifmt outp ofmt mode) = do
     program <- (getProgram ifmt inp :: IO (UplcProg PLC.AlexPosn))
     writeProgram outp ofmt mode program
 
+runDumpModel :: IO ()
+runDumpModel = do
+    let params = fromJust PLC.defaultCostModelParams
+    BSL.putStr $ Aeson.encode params
```
Comment (kwxm) [inline plutus-core/executables/uplc/Main.hs:225]: Do we maybe want to format the output more readably?  At the moment it prints 
```
{"addInteger-cpu-arguments-intercept":197209,"addInteger-cpu-arguments-slope":0,"addInteger-memory-arguments-intercept":1,"addInteger-memory-arguments-slope":1,"appendByteString-cpu-arguments-intercept":396231,"appendByteString-cpu-arguments-slope":621,"appendByteString-memory-arguments-intercept":0,"appendByteString-memory-arguments-slope":1,"appendString-cpu-arguments-intercept":150000,...
```
Maybe something like 
```
addInteger-cpu-arguments-intercept: 197209
addInteger-cpu-arguments-slope: 0
addInteger-memory-arguments-intercept: 1
addInteger-memory-arguments-slope: 1
appendByteString-cpu-arguments-intercept: 396231
appendByteString-cpu-arguments-slope: 621
...
```
would be more readable.  I suppose it depends on what we want to do with the output though.

Comment (kwxm) [reply plutus-core/executables/uplc/Main.hs:225]: It's also slightly annoying that it mixes up the CEK parameters and the builtin parameters, but I suppose that's what's in the map.

**Review by kwxm — APPROVED** [review]
I was going to ask if we should also have a command to go the other way, but that'd be easy enough to add if we ever need it.

File: plutus-core/plutus-core.cabal [hunk 2]
```diff
@@ -446,8 +446,10 @@ executable uplc
         Parsers
     build-depends:
         plutus-core -any,
+        aeson -any,
         base <5,
         bytestring -any,
+        containers -any,
```
Comment (kwxm) [inline plutus-core/plutus-core.cabal:452]:
```suggestion
```

I wondered why we needed that, but in fact it's making the build fail.  I guess it'll need `updateMaterialized` too

File: nix/pkgs/haskell/materialized-linux/.plan.nix/plutus-core.nix [hunk 3]
```diff
@@ -347,8 +347,10 @@
         "uplc" = {
           depends = [
             (hsPkgs."plutus-core" or (errorHandler.buildDepError "plutus-core"))
+            (hsPkgs."aeson" or (errorHandler.buildDepError "aeson"))
             (hsPkgs."base" or (errorHandler.buildDepError "base"))
             (hsPkgs."bytestring" or (errorHandler.buildDepError "bytestring"))
+            (hsPkgs."containers" or (errorHandler.buildDepError "containers"))
```
Comment (kwxm) [inline nix/pkgs/haskell/materialized-linux/.plan.nix/plutus-core.nix:353]:
```suggestion
            (hsPkgs."containers" or (errorHandler.buildDepError "containers"))
```

File: plutus-core/plutus-core.cabal (hunk 2 above)
Comment (kwxm) [reply plutus-core/plutus-core.cabal:452]: Wait, I can just update the nix file directly.  Let me try that.

File: nix/pkgs/haskell/materialized-darwin/.plan.nix/plutus-core.nix [hunk 4]
```diff
@@ -347,8 +347,10 @@
         "uplc" = {
           depends = [
             (hsPkgs."plutus-core" or (errorHandler.buildDepError "plutus-core"))
+            (hsPkgs."aeson" or (errorHandler.buildDepError "aeson"))
             (hsPkgs."base" or (errorHandler.buildDepError "base"))
             (hsPkgs."bytestring" or (errorHandler.buildDepError "bytestring"))
+            (hsPkgs."containers" or (errorHandler.buildDepError "containers"))
```
Comment (kwxm) [inline nix/pkgs/haskell/materialized-darwin/.plan.nix/plutus-core.nix:353]:
```suggestion
```

File: plutus-core/executables/uplc/Main.hs (hunk 1 above)
Comment (michaelpj) [reply plutus-core/executables/uplc/Main.hs:225]: I originally did that, but I anticipate that people will want to process it, so a standard format is nice. Getting it pretty-printed is a matter of piping it through `jq`

**Pull request merged**